LEJATSZO.CPP

```diff
@@ -732,8 +712,6 @@ static int Elozoshowkep1 = 1, Elozoshowkep2 = 1;
 
 // Termeszetes befejezes eseten 0-t ad vissza:
 long lejatszo_r(char* filenev, int showkepmarad) {
-    Kitoltestmegrak = 3; // kirajz meg 3x kirakja keretet
```

Comment:
This one is a bit more confusing since it's a global, but it does seem like the value of 3 isn't used before we get to kirajzol320()..